_test/test_data_loaders/test_loader_nxspe.m

```diff
@@ -470,8 +478,20 @@ function test_loader_nxspe_defines(obj)
             assertEqual({'S','ERR','en','efix','psi','det_par','n_detectors','n_det_in_par'},fields);
         end
 
+        function test_runid_from_nxspe(obj)
+            dat_file =f_name(obj,'MAP11014v3.nxspe');
+            lx = loader_nxspe(dat_file);
+            assertEqual(lx.run_id,1104)
+        end
+        
+        function test_runid_from_filename(obj)
+            dat_file =f_name(obj,'MAP11014.nxspe');
+            lx = loader_nxspe(dat_file);
+            assertEqual(lx.run_id,11014)
+        end
+
         function test_load(obj)
-            dat_file =f_name(obj,'MAP11014v2.nxspe');
+            dat_file =f_name(obj,'MAP11014v3.nxspe');
```

Comment:
here and in the other tests it would be useful to be able to change the filename at a single point, to save a lot of inspecting changes